build/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.6
```

Comment:
We should add the license header here as well to be consistent with how we do it in Pravega:

https://github.com/pravega/pravega/blob/master/docker/pravega/Dockerfile#L1

Reply:
Added.